### PR TITLE
Add xdebug documentation for vscode

### PR DESCRIPTION
**Pull request by janette**

### CODE REVIEW

File: docs/testing-debug-phpunit.md [hunk 1]
```diff
@@ -129,6 +129,32 @@ Congratulations, you have configured the IDE to run the tests.
...
+- Initialize the project for PHP Debugging by selecting “create a launch.json file” and then “PHP” from the Select Debugger options. This action will create the .vscode/launch.json file.
+- Open the launch.json and set the port and the pathMappings as follows:
+
+```
+"configurations": [
+    {
+      "name": "Listen for Xdebug",
+      "type": "php",
+      "request": "launch",
+      "port": 9003,
```
Comment (rfay) [inline docs/testing-debug-phpunit.md:145]:
```suggestion
      "hostname": "0.0.0.0",
      "port": 9003,

```

**Review by rfay — CHANGES_REQUESTED** [review]
Wouldn't it be better overall to just point to the DDEV step-debugging docs at https://ddev.readthedocs.io/en/latest/users/debugging-profiling/step-debugging/ ?

Anyway, if not, don't forget the hostname setting, which matters lots in some contexts.

File: docs/testing-debug-phpunit.md (hunk 1 above)
Comment (janette) [reply docs/testing-debug-phpunit.md]: Thank you @rfay I didn't even look to see if this existed already

**Review by rfay — APPROVED** [review]
Looks great to me, thanks!

**Comment by paul-m** [comment]
Merging this documentation change despite test fails, because documentation. :-)

**Pull request merged**